hardhat.config.ts

```diff
@@ -1,16 +1,19 @@
-require("@nomicfoundation/hardhat-toolbox");
+import { HardhatUserConfig } from "hardhat/config";
+import "@nomicfoundation/hardhat-toolbox";
+import "@nomiclabs/hardhat-ethers"
```

Comment:
This shouldn't be necessary if the toolbox is included. Does something stop working if you don't import it?

Reply:
You are right, `import "@nomiclabs/hardhat-ethers"` is redundant. Everything works as expected without it.
I remember having some trouble getting the imports right, so this was likely an artifact of that.